.eslintignore

```diff
@@ -0,0 +1 @@
+**/node_modules/**/*.js
```

Comment:
for eslint when your examples had `node_modules` in them, ideally should not happen